tests/unit/s2n_tls13_support_test.c

```diff
@@ -141,7 +141,7 @@ int main(int argc, char **argv)
         s2n_extension_type_list *tls13_server_hello_extensions = NULL;
         EXPECT_SUCCESS(s2n_extension_type_list_get(S2N_EXTENSION_LIST_SERVER_HELLO_TLS13, &tls13_server_hello_extensions));
         EXPECT_NOT_NULL(tls13_server_hello_extensions);
-        EXPECT_EQUAL(tls13_server_hello_extensions->count, 4);
+        EXPECT_TRUE(tls13_server_hello_extensions->count > 0);
```

Comment:
why not specify an exact value here. its one counter and would help verify we dont accidentally disable an extension.

Reply:
That's not what this test is testing. "Server does not parse TLS 1.3 extensions unless TLS 1.3 negotiated". By my reading, it just needs at least one extension that can be parsed / not parsed so that [this loop](https://github.com/aws/s2n-tls/blob/5981eca31e4d38c146feeed2178a6302d5d33459/tests/unit/s2n_tls13_support_test.c#L155) doesn't make the test a no-op.

There's also not really an "accidentally disable" here. tls13_server_hello_extensions is a harcoded list, and the only additions / deletions are via code changes. Like [removing the cookie extension.](https://github.com/aws/s2n-tls/pull/3306/files/5981eca31e4d38c146feeed2178a6302d5d33459#diff-381ac009a24413ee2fa3ff37c3fa2c5e6484dadd37598e97216501712073ef30)